docs/misc.rst

```diff
@@ -0,0 +1,76 @@
+.. _misc:
+
+Misc
+----
+
+Help Screens
+^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   bash-it show aliases        # shows installed and available aliases
+   bash-it show completions    # shows installed and available completions
+   bash-it show plugins        # shows installed and available plugins
+   bash-it help aliases        # shows help for installed aliases
+   bash-it help completions    # shows help for installed completions
+   bash-it help plugins        # shows help for installed plugins
+
+Pass function renamed to passgen
```

Comment:
I think we might remove this old note... It has been around for a while, not sure whether it's still needed.